aws_s3_bucket.tf

```diff
@@ -29,13 +33,17 @@ resource "aws_s3_bucket" "s3BucketSseRulesWithKmsNull" {
       }
     }
   }
+
+  versioning {
+    enabled = true
+  }
 }
 
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should have versioning enabled**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites
 
**Remediation**
S3 bucket should have versioning enabled

source: accurics-